src/DevAdventCalendarCompetition/DevAdventCalendarCompetition.Services/GoogleCalendarService.cs

```diff
@@ -53,7 +56,15 @@ private async Task<HttpResponseMessage> CreateCalendar()
             {
                 Summary = this._calendarSettings.Summary
             };
-            return await this._httpClient.PostAsJsonAsync("calendars", newCalendar);
+
+            using (var content = new StringContent(JsonConvert.SerializeObject(newCalendar), Encoding.UTF8, MediaTypeNames.Application.Json))
+            {
+                var uri = new System.Uri("https://calendar.google.com/calendars/");
```

Comment:
previously it was only "calendars" - why it is not "calendars" now?

if we need that url - please use url from settings

Reply:
We have the correct base URI in that extension that is adding HttpClient.
https://github.com/DevAdventCalendar/DevAdventCalendar/blob/develop/src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Extensions/StartupExtensions.cs#L94